### ISSUE TITLE
digitalocean_spaces_bucket NoSuchBucket while creating bucket 

### ISSUE THREAD

**Issue by blinkinglight** [digitalocean/terraform-provider-digitalocean#1006]
code in module
```
resource "digitalocean_spaces_bucket" "site-bucket" {
  name   = "randomtext-testing-${replace(var.domain, ".", "-")}"
  region = "ams3"
  acl    = "private"
}
```
modules.tf
```
module "bucket" {
  count     = 1
  source    = "./modules/bucket"
  domain = var.domain
}

```
```
│ Error: NoSuchBucket: 
│ 	status code: 404, request id: , host id: 
│ 
│   with module.bucket[0].digitalocean_spaces_bucket.site-bucket,
│   on modules/bucket/bucket.tf line 1, in resource "digitalocean_spaces_bucket" "site-bucket":
│    1: resource "digitalocean_spaces_bucket" "site-bucket" {

```

sometimes it fails sometimes not

**Comment by blinkinglight** [comment]
is there any date planned to release this change? https://github.com/digitalocean/terraform-provider-digitalocean/pull/999 or maybe dev/nightly builds exists?

**Comment by andrewsomething** [comment]
That change was included in the v2.29.0 release. If this did not resolve the problem for you, please feel free to reopen this issue.

https://github.com/digitalocean/terraform-provider-digitalocean/releases/tag/v2.29.0

**Comment by srasul** [comment]
Still happening with v2.34.1
